content/en/docs/concepts/scheduling-eviction/kube-scheduler.md

```diff
@@ -66,6 +66,11 @@ resource requests. After this step, the node list contains any suitable
 Nodes; often, there will be more than one. If the list is empty, that
 Pod isn't (yet) schedulable.
 
+The scheduler also respects [node conditions](/docs/concepts/scheduling-eviction/pod-eviction#node-conditions)
+when making scheduling decisions. For example, if the `MemoryPressure` node condition is active on a node, the scheduler
+does not schedule new `BestEffort` pods on the node. If the `DiskPressure` node
+condition is active, the scheduler does not schedule any new pods on the node. 
```

Comment:
Well, the fact that the admission plugin doesn't add a toleration means that nodes that have the taint would get no pods scheduled. The special case is rather `MemoryPressure`, which has an explicit toleration for non `BestEffort` pods.

Now.... it's not really the scheduler which implements this. It's the apiserver. We can add this detail in https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition, which already explains how taints and node conditions are related.

Reply:
So in this case, remove the `DiskPressure` part?

We should create a new issue to update Taints and Tolerations as well.

Reply:
The statement is correct, but I would change the order.

> For example, If the `DiskPressure` node condition is active, the system doesn't allow any new Pods to schedule on the node. If the `MemoryPressure` node condition is active on a node, the system doesn't allow new `BestEffort` pods on the node.

BUT, I wouldn't have this in under the kube-scheduler page (because it's not something kube-scheduler does). It should be in the taints and tolerations doc.

Reply:
Opened #27878 for this.

Reply:
Can you remove this paragraph from here then?

Reply:
Yup already removed. I'm holding off from pushing until other comments are resolved. Anything I resolve here is fixed in the working copy :)